flixel/FlxGame.hx

```diff
@@ -243,6 +243,14 @@ class FlxGame extends Sprite
 		// Super high priority init stuff
 		_inputContainer = new Sprite();
 		
+		if (GameSizeX == 0)
+		{
+			GameSizeX = Std.int(Lib.current.stage.window.width);
```

Comment:
I don't think that matters? `get_stage()` is just a shortcut for `Lib.current.stage`.